webui/react/src/components/TaskActionDropdown.tsx

```diff
@@ -42,21 +33,26 @@ const TaskActionDropdown: React.FC<Props> = ({
 
   const confirm = useConfirm();
 
-  const handleMenuClick = (params: MenuInfo): void => {
-    params.domEvent.stopPropagation();
+  const menuItems: MenuItem[] = [];
+
+  if (isKillable) menuItems.push({ key: Action.Kill, label: 'Kill' });
+
+  menuItems.push({
+    key: Action.ViewLogs,
+    label: <Link path={paths.taskLogs(task as CommandTask)}>View Logs</Link>,
+  });
```

Comment:
nit: if this is always going to be in `menuItems`, why not just put it in during initialization? If you're concerned about the order you can just unshift Kill